src/get-repos.js

```diff
@@ -3,15 +3,17 @@ module.exports = async function (args, octokit) {
     return [args.repo];
   }
 
-  let repos;
+  let repos = [];
   if (args.org) {
-    ({ data: repos } = await octokit.repos.listForOrg({
-      org: args.org,
-    }));
+    for await (const response of octokit.paginate.iterator(octokit.repos.listForOrg, { org: args.org })) {
```

Comment:
Rather than iterating over each entry, you can use the second parameter from `octokit.paginate` (see https://github.com/mheap/github-default-branch/blob/main/bin/github-default-branch#L87-L96)

```javascript
let repos = await octokit.paginate(
  octokit.repos.listForOrg,
  {
    org: args.org
  },
  (response) => response.data
);
```